Keas.Mvc/Startup.cs

```diff
@@ -142,7 +142,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
                     name: "API",
                     template: "api/{teamName}/{controller}/{action}/{id?}",
                     defaults: new { controller = "people", action = "Index" },
-                    constraints: new { controller = "(keys|equipment|access|spaces|people|person|workstations)" }
+                    constraints: new { controller = "(keys|equipment|access|spaces|people|person|workstations|tags)" }

```

Comment:
@laholstege I worry that this will mess up the existing tags controller, so I'm definitely thinking now we just move the tag lookup part to some new controller so we don't get conflicts